### PR TITLE
chore: dependency lifecycle, makefile updates

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -12,7 +12,6 @@ func main() {
 	config := &api.GoWRK2Config{
 		Thread:            1,
 		DurationInSeconds: 5,
-		// Connection:        10,
```
Comment (leecalcote) [inline main.go:15]: I'd like to leave this in as a reminder of what we could be using.

File: main.go [hunk 2]
```diff
@@ -12,7 +12,6 @@ func main() {
...
 		RQPS:        10,
 		URL:         "http://gmail.com",
```
Comment (leecalcote) [inline main.go]: Change to https://meshery.io?

File: main.go [hunk 3]
```diff
@@ -12,7 +12,6 @@ func main() {
 	config := &api.GoWRK2Config{
 		Thread:            1,
 		DurationInSeconds: 5,
```
Comment (leecalcote) [inline main.go]: Just for consistency (not that it is material), change this to 30?

**Comment by kushthedude** [comment]
@leecalcote fixed

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**